controllers/articleController.js

```diff
@@ -0,0 +1,15 @@
+const { getArticleById } = require('../models/articleModel');
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  // check that article id is an integer, if it is not, return an error
+  if (!Number.isInteger(Number(article_id))) {
+    return res.status(400).send({ msg: 'Bad Request - Article ID was invalid.' });
+  }
+
```

Comment:
You already added an error handler for PSQL errors that you are not using - personally, I would say change your test to the error message used in your PSQL error handling & delete this, makes for cleaner code :)

Reply:
I took this approach to protect against SQL injection